docs/content/users/usage/networking.md

```diff
@@ -177,6 +178,34 @@ If your file contains this header, renaming it to `.crt` is sufficient for use w
 | Rancher Desktop  | Yes (dockerd)        | Partial                   | Depends on Lima config and backend     |
 | WSL2 + docker-ce | Yes                  | 🚫                        | Must configure trust inside WSL        |
 
+### Cloudflare WARP (Zero Trust)
+
+If you are using Cloudflare WARP (Zero Trust), you might find that `*.ddev.site` domains do not resolve because WARP intercepts and routes all traffic through its network, bypassing your local loopback.
```

Comment:
```suggestion
If you are using [Cloudflare WARP](https://developers.cloudflare.com/cloudflare-one/team-and-resources/devices/warp/) (Zero Trust), you might find that `*.ddev.site` domains (or other domains you might use that resolve to 127.0.0.1) do not resolve because WARP intercepts and routes all traffic through its network, bypassing your local loopback.
```